test/e2e/config/caren.yaml

```diff
@@ -134,8 +134,8 @@ providers:
 - name: caren
   type: RuntimeExtensionProvider
   versions:
-  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.11}"
-    value: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/download/{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.11}/runtime-extension-components.yaml"
+  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.13}"
```

Comment:
Can you please also update https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/813/files#diff-dc2da7b81e3e98ccae5c455c11d2104e2eca318ab838d04f73e65d8e1a1eee8cR148 to `v0.14.99` for the next version to build and test?